visualization/app/codeCharta/ui/copyToClipboardButton/util/getFilenamesWithHighestMetrics.md

```diff
@@ -0,0 +1,58 @@
+## More perfomant alternativ for insertion algorithm
```

Comment:
Please copy past this text into the PR or into the issue and delete this file. I am not confident this documentation will be useful in half a year within the codebase.

Reply:
Applied with commit [4045391](https://github.com/MaibornWolff/codecharta/pull/2942/commits/40453916f006e10ae233021e376bd307576a0f54)